src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final double length;
+    private final double width;
+
+    public Rectangle(String color, double length, double width) {
+        super(color);
+        this.length = length;
+        this.width = width;
+    }
+
+    @Override
+    public double getArea() {
+        return length * width;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle,"
+                + " area: " + getArea()
+                + " sq. units, length: "
+                + length + " units, width:"
+                + width + " units, color: " + super.getColor());
```

Comment:
In the draw method, instead of concatenating strings, it's better to use String.format for readability and performance reasons.